### ISSUE TITLE
Create dataset page

### ISSUE THREAD

**Issue by xavijam** [CartoDB/cartodb#7950]
---

- [x] Render table info view.
- [x] Disable (readonly) if it is sync.
- [x] Render table view.

---

- [x] Move notifier component (https://github.com/CartoDB/cartodb/pull/8321).
- [x] Added notifier.
- [x] Rename table.
- [x] Rename table confirmation.
- [x] Add menu bar.
- [x] Disable (readonly) for users without permission.
- [x] Fix css problems.
- [x] Disable state.
- [x] Tests.

---

- [x] Duplicate table
- [x] Delete table
- [x] Lock dataset
- [x] Unlock dataset
- [x] Check table scroll problems.
- [x] Create map from layer modal and behaviour.
- [x] Add row and column. 
- [x] Tests

---

- [x] Add sync info.
- [x] Edit sync options modal.
- [x] Add SQL editor.
- [x] Tests.

---

- [x] Migrate table-confirmation-modal to confirmation-modal created by @nobuti.
- [x] Privacy modal.
- [x] Export dialog -> branch: export-modal

---

- [x] Add map preview.
- [x] Check styles (@piensaenpixel).

---

- [ ] Add table metadata info.
- [ ] Edit table metadata modal.
- [ ] Share info.
- [ ] Check browserify vendors, their size is too much for the table.

**Comment by saleiva** [comment]
:*

**Comment by javitonino** [comment]
> There are only two hard things in Computer Science: cache invalidation and naming things.
> Phil Karlton

About naming the URLs:
- `/tables/name` is taken by the old table view. We could redirect based on feature flag, but it would be ugly.
- `/datasets/name` is free, but we already have `/datasets`, `/datasets/page/#` and `/datasets/tag/:tag` for public dataset listing. It will be difficult to grow, e.g: `/datasets/name/public` would be ambiguous for tables named `tag`.
- `/dataset/name` & `/table/name` violate our naming policy until now (all plurals).
- `/builder/datasets|tables/name` ? is maybe too much nesting

So basically, we are running out of names to use. Ideas? I guess it depends a bit on future plans for public table view.

cc/ @javisantana @juanignaciosl

**Comment by juanignaciosl** [comment]
IMHO `/builder/<name>/table` or `/dataset/<name>`.

**Comment by javitonino** [comment]
Merged and deploying. `/dataset/table_name` or `/dataset/canonical_visualization_id` are valid.

**Comment by xavijam** [comment]
First step merged

**Comment by xavijam** [comment]
To be reviewed in styles @piensaenpixel:
- Everything in general :joy:
- Date edit field.
- SQL label when custom query is applied.
- Rows with empty values appear weird. 
![screen shot 2016-07-06 at 12 14 23](https://cloud.githubusercontent.com/assets/132146/16614741/38430e64-4373-11e6-9a3c-ce259c67c4d6.png)

**Comment by xavijam** [comment]
Moved last pending issues to new ones, so closing.
